README.md

```diff
@@ -13,6 +13,9 @@ The Puppet master is configured under Apache and Passenger by default, unless
 3.x based installation, `server_implementation` can be set to `puppetserver`
 to switch to the JVM-based Puppet Server.
 
+When using Puppet Server 2 (version 2.0 was the first version to support Puppet 4),
+Puppet Server 2.2 is the minimum version supported by this module.
```

Comment:
Am I understanding the situation correctly that for puppet server >= 2.3 you need to set ```server_use_legacy_auth_conf``` to ```true``` and for >= 2.0 < 2.3 to ```false```?